src/async_fuse/test/integration_tests.rs

```diff
@@ -134,7 +134,7 @@ fn test_directory_manipulation_rust_way(mount_dir: &Path) -> anyhow::Result<()>
 }
 
 #[cfg(test)]
-fn test_deferred_deletion(mount_dir: &Path) -> anyhow::Result<()> {
+fn _test_deferred_deletion(mount_dir: &Path) -> anyhow::Result<()> {
```

Comment:
Mark `#[allow(dead_code)]` instead of adding `_` as prefix.

Reply:
And add some comment to describe why this test case is disabled.

Reply:
I will update it.